src/internal_modules/roc_packet/target_libuv/roc_packet/udp.h

```diff
@@ -37,6 +37,11 @@ struct UDP {
     uv_udp_send_t request;
 
     UDP();
+
+    //! It points to a moment when the packet was transferred to a sink-thread, that
+    //! "consumes" this packet. The reason to have it separate is that this allows
+    //! us to account additional jitter introduced by thread-switch time.
+    core::nanoseconds_t enqueue_ts;
```

Comment:
1. We need to initialize it to zero in udp.cpp.

1. Please mention in comment that it's nanoseconds since unix epoch.

1. Could you place it near receive_timestamp above please?

1. What about "queue_ts" instead of "enqueue_ts"? Then we can use nice acronym QTS for it (easily distinguishable compared to e.g. "ETS").

1. Could you also document it here? https://github.com/roc-streaming/roc-toolkit/blob/develop/docs/sphinx/internals/timestamps.rst